test/utils/image/manifest.go

```diff
@@ -216,7 +214,8 @@ const (
 
 func initImageConfigs(list RegistryList) (map[ImageID]Config, map[ImageID]Config) {
 	configs := map[ImageID]Config{}
-	configs[Agnhost] = Config{list.PromoterE2eRegistry, "agnhost", "2.56"}
+	configs[Agnhost] = Config{list.PromoterE2eRegistry, "agnhost", "2.55"}
+	configs[AgnhostNew] = Config{list.PromoterE2eRegistry, "agnhost", "2.56"}
```

Comment:
lets keep the latest version mapped to `Agnhost` and rename `AgnhostNew` to something else to point to the older version.

Reply:
Is there any available doc recording recent usable agnhost image versions?

Reply:
well, its already pointing to the latest one(2.56) and you can find all the tags here - https://explore.ggcr.dev/?repo=registry.k8s.io%2Fe2e-test-images%2Fagnhost

Reply:
The `agnhost` image name has been changed, and the PR passed all testing jobs except unrelated flaky cases before squashing. Please take a look.

Reply:
if we follow @mkumatag suggestion IIUIC that has also the benefit we do not need to change any of the existing references to the `Agnhost` image , that sounds really nice, the less things we need to change the better

Reply:
> if we follow @mkumatag suggestion IIUIC that has also the benefit we do not need to change any of the existing references to the `Agnhost` image , that sounds really nice, the less things we need to change the better

The change is not obvious to be launched. I'll take some time to find out tests not related to in-place image update and revert the `AgnhostImagePrev` changes to `AgnhostImage`